### PR TITLE
feat: Add troubleshooting guide for user plane issues

### PR DESCRIPTION
**Pull request by ghislainbourgeois**
# Description

This adds a troubleshooting guide for user plane issues. Because it is impossible to know all the issues in advance, the guide gives a bit of explanation throughout to help the reader apply the commands properly for their issue.

This will probably need a refactoring once we have a reference guide on BESS.

# Checklist:

- [x] Documentation follows the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: docs/how-to/troubleshoot_user_plane_issues.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+expose the port to the outside:
+
+```shell
+kubectl port-forward -n <model_name> --address 0.0.0.0 pod/upf-0 5000
+```
+
+Connect to the server from your browser on port 5000. The URL should be
+similar to `http://<k8s-node-ip>:5000`.
+
+You should a graph similar to this on the page:
```
Comment (Gmerold) [inline docs/how-to/troubleshoot_user_plane_issues.md:149]:
```suggestion
You should see a graph similar to this on the page:
```

**Review by Gmerold — APPROVED** [review]
One minor comment.
I wish we had that earlier :D

File: docs/how-to/troubleshoot_user_plane_issues.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
+# Troubleshoot user plane issues
+
+This guide provides step-by-step troubleshooting actions to identify user plane
+issues. It assumes that a UE (User Equipḿent) was able to connect to the network
+and establish a PDU session, meaning that the UE received an IP address. It also
+assume that the UE already tried using the connection unsuccessfully.
```
Comment (gatici) [inline docs/how-to/troubleshoot_user_plane_issues.md:6]:
```suggestion
assumes that the UE already tried using the connection unsuccessfully.
```

File: docs/how-to/troubleshoot_user_plane_issues.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+5. Packets will go out of the UPF by going through `coreNAT` and `coreQ0FastPO`
+
+Then, trace the packets coming back from the data network:
+
+1. Start at the `coreQ0FastPI` node
+2. Packets should go through the `coreNAT` node
+3. Packets should then go through the `gtpuEncap` node
+3. Packets should make their way to the `accessRoutes` node
+4. From the `accessRoutes` node, they should go to the `accessDstMAC*` node
+5. Packets will go out of the UPF by going through `accessQ0FastPO`
```
Comment (gatici) [inline docs/how-to/troubleshoot_user_plane_issues.md:180]:
```suggestion
4. Packets should make their way to the `accessRoutes` node
5. From the `accessRoutes` node, they should go to the `accessDstMAC*` node
6. Packets will go out of the UPF by going through `accessQ0FastPO`
```

**Review by gatici — COMMENTED** [review]
Very useful guide. Thanks!

**Pull request merged**